### DISCUSSION TITLE
Faster way to connect many to one / one to many

### DISCUSSION THREAD

**Discussion by nordcart** [directus/directus#8281] [Feature Requests]
1. Start to write the first letters of the connection, matches appears below.
2. Select the connection.

This could be good if there are a massive list.

Screenshot from directus:
![image](https://user-images.githubusercontent.com/12813484/134646721-48e232cb-3fcf-4456-b24d-b1101b7fe918.png)

Example of how it could be:
![image](https://user-images.githubusercontent.com/12813484/134646883-ba2ecc32-1fec-4458-90e9-7c8d06c05193.png)
--------------------------
More letters in the search --> greater match
![image](https://user-images.githubusercontent.com/12813484/134646563-297e4201-594a-4bc3-8ada-636a6ad7955f.png)

**Comment by benhaynes** [comment]
I think it would be nice to have this functionality, but not on-screen at all times. I think as soon as someone clicks "Select Existing", we open the drawer (as is done now, for getting access to different Layouts), but with the search bar open and focused (can't think of a reason not to). That way, you can click "Select Existing" and immediately start typing to filter down. Seems like a nice balance.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
